src/main/java/org/graylog/aws/cloudwatch/CloudWatchLogData.java

```diff
@@ -31,6 +31,13 @@
  * </pre>
  */
 public class CloudWatchLogData {
+
     @JsonProperty("logEvents")
     public List<CloudWatchLogEvent> logEvents;
+
+    @JsonProperty("logGroup")
+    public String logGroup;
```

Comment:
Can this ever by `null` (i. e. not exist)?

Reply:
Looking at the [documentation and example](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/SubscriptionFilters.html), there is nothing to suggest that this field is optional / might not be present in any situation. Additionally, in all of my testing to date it has never not been present.

Reply:
✅